saw-remote-api/python/tests/saw-in-progress/drbg/drbg.py

```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env python3
 import os
 import os.path
-from saw import *
-from saw.llvm import Contract
-from saw.proofscript import yices
-from cryptol import cry, cryptoltypes
+from saw_client import *
+from saw_client.llvm import Contract, cry, cry_f, array_ty, alias_ty, i8, i32, i64
```

Comment:
Is it funny that `cry` and `cry_f` get imported from the `llvm` module of `saw_client`, or is that just me?

Reply:
E.g., seeing `llvm.cry` `llvm.cry_f` makes me think there is something LLVM-y about `cry` and `cry_f`.